utils/gyb_sourcekit_support/UIDs.py

```diff
@@ -38,6 +38,7 @@ def __init__(self, internal_name, external_name):
     KEY('Line', 'key.line'),
     KEY('Column', 'key.column'),
     KEY('ReceiverUSR', 'key.receiver_usr'),
+    KEY('Receivers', 'key.receivers'),
```

Comment:
I wasn't sure whether to keep the "USR" part or not, some of the keys include it and some don't. Thoughts? (Eg. Overrides is not OverrideUSRs)

Reply:
I prefer 'Receivers', and it'd be good not to just have it a flat list of USRs, but array of dictionaries, even if the only entry is a 'key.usr' in the beginning.
This will give us flexibility to extend it with more info in the future.

Reply:
Ah that's true, it is actually already a dictionary with key.usr so yep, adding "USRs" wouldn't make much sense!